mindmeld/components/dialogue.py

```diff
@@ -1339,6 +1345,7 @@ def process(self, text, params=None, force_sync=False):
             context=self.context,
             frame=self.frame,
             history=self.history,
+            verbose=self.verbose
```

Comment:
Change looks good. Can you also add the verbose flag here: https://github.com/cisco/mindmeld/pull/164/files#diff-c21e7ac893d7b0e1d68df2df2ce7ff52R1389

Reply:
Sorry, I'm not quite sure which line you're referring to. The link doesn't do anything when I click it. Can you please clarify?

Reply:
I mean line 1377 (old code) and 1384 (new code) of `mindmeld/components/dialogue.py`: 
```
        response = await self._app_manager.parse(
            text,
            params=internal_params,
            context=self.context,
            frame=self.frame,
            history=self.history,
        )
```

Reply:
Done! Thank you for the clarification.